hugo/layouts/lookout/single.html

```diff
@@ -91,4 +111,4 @@ <h3>source{d} Lookout</h3>
 {{ partial "try-sourced.html" }}
 {{ partial "footer.html" }}
 </body>
-</html>
+</html>
```

Comment:
All files should end with an empty line :dagger: 
```suggestion
</html>

```